src/DataFramesMeta.jl

```diff
@@ -207,18 +219,72 @@ Multiple `i` expressions are "and-ed" together.
 
 ### Examples
 
-```julia
-df = DataFrame(x = 1:3, y = [2, 1, 2])
-x = [2, 1, 0]
-
-@where(df, :x .> 1)
-@where(df, :x .> x)
-@where(df, :x .> x, :y .== 3)
-
-d = DataFrame(n = 1:20, x = [3, 3, 3, 3, 1, 1, 1, 2, 1, 1, 2, 1, 1, 2, 2, 2, 3, 1, 1, 2])
-g = groupby(d, :x)
-@where(d, :x .== 3)
-@where(g, length(:x) > 5))   # pick out some groups
+```jldoctest
+julia> using DataFramesMeta, DataFrames
+
+julia> df = DataFrame(x = 1:3, y = [2, 1, 2]);
+
+julia> x = [2, 1, 0];
+
+julia> @where(df, :x .> 1)
+2×2 DataFrames.DataFrame
+│ Row │ x │ y │
+├─────┼───┼───┤
+│ 1   │ 2 │ 1 │
+│ 2   │ 3 │ 2 │
+
+julia> @where(df, :x .> x)
+2×2 DataFrames.DataFrame
+│ Row │ x │ y │
+├─────┼───┼───┤
+│ 1   │ 2 │ 1 │
+│ 2   │ 3 │ 2 │
+
+julia> @where(df, :x .> x, :y .== 3)
+0×2 DataFrames.DataFrame
+
+julia> d = DataFrame(
+            n = 1:20,
```

Comment:
That's an unusual way of wrapping lines. I'd rather keep `n` on the first line, but split `x` over multiple lines. Same remark in the next docstring.